src/leader_election.rs

```diff
@@ -231,9 +247,15 @@ pub mod ballot_leader_election {
 
         /// Initiates a new heartbeat round.
         pub fn new_hb_round(&mut self) {
+            info!(self.logger, "Initiate new heartbeat round");
```

Comment:
```suggestion
            trace!(self.logger, "Initiate new heartbeat round");
```
Also include the round